ts/Core/Axis/TreeGridAxis.ts

```diff
@@ -902,10 +903,12 @@ namespace TreeGridAxis {
 
             return Object.keys(axis.treeGrid.mapOfPosToGridNode || {}).reduce(
                 function (arr: Array<number>, key: string): Array<number> {
-                    var pos = +key;
+                    var pos = +key,
+                        roundedMin = Math.floor(axis.min / axis.tickInterval) * axis.tickInterval,
+                        roundedMax = Math.ceil(axis.max / axis.tickInterval) * axis.tickInterval;
```

Comment:
Instead of defining in `reduce()`, could we define in in the higher scope? To calculate this just once.

Reply:
That is a good point. I will change in the next commit.